tests/DynamicEffects_Tests.cpp

```diff
@@ -0,0 +1,91 @@
+/**
+ * @file
+ * @brief Unit tests for openshot::Timeline
+ * @author Jonathan Thomas <jonathan@openshot.org>
+ *
+ * @ref License
+ */
+
+/* LICENSE
+ *
+ * Copyright (c) 2008-2019 OpenShot Studios, LLC
+ * <http://www.openshotstudios.com/>. This file is part of
+ * OpenShot Library (libopenshot), an open-source project dedicated to
+ * delivering high quality video editing and animation solutions to the
+ * world. For more information visit <http://www.openshot.org/>.
+ *
+ * OpenShot Library (libopenshot) is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU Lesser General Public License
+ * as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * OpenShot Library (libopenshot) is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with OpenShot Library. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "UnitTest++.h"
+#include "../include/OpenShot.h"
+
+#ifdef TEST_PLUGIN
+#if defined(__linux__)
```

Comment:
I was going to suggest combining these as
```cpp
#if defined(__linux__) && defined(TEST_PLUGIN)
```
But since in theory we might have plugin-loading implementations for other OSes, leaving them separate may be better. The only thing I'd request is commenting the `#endif`s as:
```cpp
#endif // defined(__linux__)
#endif // TEST_PLUGIN
```
since they're way too far away to match up visually.